# No exception is raised
assert 1 == 1

# An exception is raised
assert 1 == 2
